config_hub/finetune/README.md

```diff
@@ -0,0 +1,10 @@
+## Config files
+
+The table below lists the performances you can expect from the provided config files. Note that you can achieve lower memory consumption by lowering the micro batch size as needed. See the [Dealing with out-of-memory (OOM) errors](../../litgpt/tutorials/oom.md) on lowering the memory requirements.
+
+|                       | Size | Dataset   | Epochs | Val loss | Peak memory | Max seq length | Micro batch size | Precision | Training runtime |
+| --------------------- | ---- | --------- | ------ | -------- | ----------- | -------------- | ---------------- | --------- | ---------------- |
```

Comment:
Please add the new configs to https://github.com/Lightning-AI/litgpt/blob/wip/tests/test_config_hub.py

Reply:
Done

Reply:
Missing qlora!

Reply:
sorry I had some issues with QLoRA tests and just temporarily removed them; will add them back